tests/nnstreamer_grpc/runTest.sh

```diff
@@ -67,6 +67,25 @@ export LD_LIBRARY_PATH=${PATH_TO_PLUGIN_EXTRA}:${LD_LIBRARY_PATH}
 
 NUM_BUFFERS=10
 
+TIMEOUT_SEC=10
+## Check if a pipeline is launched and playing
+function waitformarker {
```

Comment:
Q: is it possible to add util script and use common function waitformarker (query and grpc)?

Reply:
It'd be better located at SSAT (ssat-api). Because it's in a different repo, let's do it later